src/main/java/com/echobox/api/linkedin/util/ValidationUtils.java

```diff
@@ -115,10 +121,14 @@ public static void validateRequiredResponseHeader(Map<String, String> headers, S
    *
    * @param fileSizeBytes size of the video file in bytes
    */
-  public static void validateVideoFileSize(long fileSizeBytes) {
-    if (fileSizeBytes > Integer.MAX_VALUE) {
+  public static void validateVideoFileSize(Long fileSizeBytes) {
```

Comment:
It's a bit weird to use an object Long in this validation function. As we want to have the functionality asap, it's fine for now.
Please raise a tech debt ticket to fix it later (together with the 2GB limit)

Reply:
I made it an object because it's possible that the field is not set in the request object. If that's the case then the primitive value would be zero and pass all the tests